src/gpio.rs

```diff
@@ -464,13 +474,16 @@ where
         let gpio = unsafe { &*pac::GPIO::ptr() };
         let registers = Registers::new(gpio);
 
-        registers.not[T::PORT].write(|w| unsafe { w.notp().bits(T::MASK) });
+        let port = self.inner().port();
+
+        registers.not[port]
+            .write(|w| unsafe { w.notp().bits(self.inner().mask()) });
```

Comment:
Another tiny nitpick: This is somewhat inconsistent with similar code below.

```suggestion
        registers.not[self.inner().port()]
            .write(|w| unsafe { w.notp().bits(self.inner().mask()) });
```